static/app/views/monitors/components/monitorQuickStartGuide.tsx

```diff
@@ -108,6 +110,16 @@ const onboardingGuides: Record<string, OnboardingGuide> = {
       'java-spring',
     ]),
   },
+  ruby: {
+    label: 'Ruby',
+    Guide: RubyCronQuickStart,
+    platforms: new Set(['ruby']),
+  },
+  rubyRails: {
+    label: 'Rails',
+    Guide: RubyRailsCronQuickStart,
+    platforms: new Set(['ruby-rails']),
```

Comment:
should we also allow this to be shown for `ruby` or is it uncommon for a developer to select `ruby` as their project platform when they really have a ruby on rails application?

Reply:
yep will add, wasn't really sure what this `platforms` list affects